e2e-tests/Jenkinsfile

```diff
@@ -15,12 +15,8 @@ pipeline {
     timeout(time: 1, unit: 'HOURS')
   }
   environment {
-    PYTEST_ADDOPTS =
-      "-n=10 " +
-      "--color=yes " +
-      "--tb=short " +
-      "--driver=SauceLabs " +
-      "--variables=capabilities.json"
+    /** See https://issues.jenkins-ci.org/browse/JENKINS-42857 - we'd like to expand this out into multi-line concatenations */
```

Comment:
Looks like you already raised this as https://issues.jenkins-ci.org/browse/JENKINS-42771. Just a note for other reviewers, this issue is a regression in version 1.1.1 of the Pipeline Model Definition Plugin. We want to upgrade to this version to take advantage of one of the new features, which either means waiting for the mentioned issue to be fixed (it doesn't sound like it will be too long), or adding this temporary workaround.